src/Models/OA/Offer.php

```diff
@@ -147,15 +147,15 @@ public static function fieldList() {
 
     /**
      * The offer price of the activity.
-     * This price should be specified without currency symbols and as a floating point number with two decimal places.
+     * This price should be specified without currency symbols and as a floating point number/string with two decimal places.
```

Comment:
```suggestion
     * This price should be specified without currency symbols and as a decimal number/string with two decimal places.
```

As "floating point" has a specific connotation here?